src/AjaxDynamicDataTable.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DynamicDataTable from "./DynamicDataTable";
+import {AxiosInstance as axios} from "axios";
+
+class AjaxDynamicDataTable extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            rows: [],
+            currentPage: 1,
+            totalPages: 1,
+            orderByField: null,
+            orderByDirection: null,
+        };
+    }
+
+    componentDidMount() {
+        this.loadPage(1);
+    }
+
+    render() {
+
+        const state = this.state;
+
+        return (
+            <DynamicDataTable
+                rows={state.rows}
+                currentPage={state.currentPage}
+                totalPages={state.totalPages}
+                orderByField={state.orderByField}
+                orderByDirection={state.orderByDirection}
+                changePage={page => this.changePage(page)}
```

Comment:
Don't use anonymous functions because of the bad performance it causes.

http://johnnyji.me/react/2016/06/27/why-arrow-functions-murder-react-performance.html

Reply:
Done.